forge-gui-mobile/src/forge/adventure/scene/RewardScene.java

```diff
@@ -318,6 +319,14 @@ void updateCollectionPool() {
     }
 
     public void loadRewards(Array<Reward> newRewards, Type type, ShopActor shopActor) {
+        // Sort the rewards based on the rarity of the card inside the reward/ lets give items rarity
+        newRewards.sort(Comparator.comparing(reward -> {
+            if (reward.getCard() != null && reward.getCard().getRarity() != null) {
```

Comment:
If we're going to be sorting them, should we also consider color or name as secondary sorts? Doesn't necessarily have to be in this PR but just a thought..

Reply:
I considered it. I thought it might be overdoing it, but in the large booster box dumps it would be best. We can accept and I'll add later.